source/scss/_pf-basic-tabs.scss

```diff
@@ -0,0 +1,52 @@
+//
+// Tabs
+//
+
+.nav-link {
+    color: $pf-tabs-color;
+}
+
+.nav-tabs {
+
+  // overrides bootstrap's spacing
+  .nav-link {
+    padding: $pf-spacer-xxxs $pf-spacer-sm;
+  }
+  // overrides bootstrap to make the tabs butt next to each other
+  .nav-item + .nav-item {
```

Comment:
Sure, I will change that, although that one I actually did leave intentionally because I thought it was clearer with the selector all together, and there were no other .nav-item styles.